tests/pipelines/eks/small/upstream-load.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: eks-small-upstream-load
+spec:
+  schedule: "0 8 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl
+            image: curlimages/curl
+            args: ["curl", "-X", "POST", "--data", "{}", "el-eks-small-upstream-load.svc.cluster.local:8080"]
```

Comment:
Curious if this uri is still correct

Reply:
yup, iiuc - https://github.com/awslabs/kubernetes-iteration-toolkit/pull/24/files#diff-9e51872dc11d990860aee6cc22d27c3920807c116aea7a4e8c8a04cf5ea51593R26